src/assets/locales/fr.json

```diff
@@ -0,0 +1,567 @@
+  "spectating-label.label": "Observer",
+  "support.copied": "copié!",
+  "support.copy": "copier",
+  "support.details": "details",
+  "support.in-app-browser-android": "Copier coller ce lien directement dans Chrome ou Firefox",
+  "support.in-app-browser-ios": "Copier et coller ce lien directement dans Safari",
+  "support.missing-features": "Certaines fonctionnalités de votre navigateur sont manquantes.",
+  "support.supported": "supporté",
+  "support.unsupported": "non supporté",
+  "support.update-browser": "S'il vous plait, essayez de changer ou mettre à jour vers un navigateur plus récent",
+  "tips.desktop.invite": "Personne d'autre est ici. Utilisez le bouton inviter en bas à gauche pour partager cette salle.",
+  "tips.desktop.locomotion": "Utilisez les touches W A S D pour bouger. Maintenez shift pour accélérer.",
+  "tips.desktop.look": "Bienvenue dans {appName}! Faisons un tour rapide. 👋 Cliquez et faites glisser pour regarder autour.",
+  "tips.desktop.turning": "Parfait. Utilisez Q et E pour tourner.",
+  "tips.discord": "Le Chat dans cette salle est relié à {broadcastTarget} sur Discord.",
+  "tips.dismiss.ok": "Ok",
+  "tips.dismiss.skip": "Passer",
+  "tips.embedded": "Cette salle est embedded, donc elle peut être visible aux visiteurs sur d'autres sites web.",
+  "tips.fullscreen": "Entrée dans le mode plein écran. Presser Echap pour montrer l'interface utilisateur.",
+  "tips.lobby": "Vous êtes dans le hall d'attente. Les autres ne peuvent pas vous voir ou vous entendre.",
+  "tips.mobile.invite": "Utiliser le bouton d'invitation en bas à gauche pour partager cette salle.",
+  "tips.mobile.locomotion": "Enorme! Pour vous déplacer, pincez avec deux doigts.",
+  "tips.mobile.look": "Bienvenue! 👋 Tapez et faites glisser pour regarder autour.",
+  "tips.streaming": "Diffusion actuelle vers le hall d'attente. Sortir du mode streamer par le menu 'plus' quand vous aurez fini.",
+  "toolbar.enter-vr-button": "Entrer en VR",
+  "toolbar.join-room-button": "Joindre la Salle",
+  "toolbar.leave-room-button": "Quitter",
+  "toolbar.spectate-in-vr-button": "Observer en VR",
+  "tweet-editor-modal.sending-tweet": "Envoi du Tweet...",
+  "tweet-editor-modal.title": "Tweet",
+  "tweet-editor-modal.tweet-button": "Tweet",
+  "twitter-oauth-modal.connect-button": "Se connecter à Twitter",
+  "twitter-oauth-modal.message": "Se connecter à twitter pour twitter depuis {appName}.",
+  "twitter-oauth-modal.title": "Se connecter à Twitter",
+  "ui-root.interstitial-prompt": "Continuer",
+  "user-profile-sidebar.demote-button": "Rétrograder",
+  "user-profile-sidebar.demote-button-disabled-label": "{displayName} est déconnecté(e).",
+  "user-profile-sidebar.hide-button": "Cacher",
+  "user-profile-sidebar.kick-button": "Ejecter",
```

Comment:
```suggestion
  "user-profile-sidebar.kick-button": "Bannir",
```

Reply:
still the same : 
kick = ejecter
ban = bannir
kick is temporary
ban is permanent